File: Frontend/vencimientos/src/app/models/bills/bill.ts
import { MonthlyClient } from "../clients/monthlyClient";

export class Bill {
  billNumber!: number;
  isActive!: boolean;
  totalAmount!: number;
  issueDate!: Date;
  paymentDate!: Date;
  monthlyClientId!: number;
  monthlyClient!: MonthlyClient;
  year!: number;
  month!: number;
  description!: string;
}

File: Backend/Domain/Bills/Bill.cs
using Domain.Clients;
using Domain.Vehicles;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Bills
{
    public class Bill
    {
        public int Id { get; set; }

        public int BillNumber { get; set; }

        public bool IsActive { get; set; }

        public int TotalAmount { get; set; }

        public DateTime IssueDate { get; set; }

        public DateTime PaymentDate { get; set; }

        //public List<Vehicle> Vehicles { get; set;}

        public int MonthlyClientId { get; set; }
        public MonthlyClient MonthlyClient { get; set; }

        public int Year { get; set; }

        public int Month { get; set; }

        public string Description { get; set; }


        public Bill() { }
    }
}
